### PR TITLE
jdk-21 Oct CPU installers

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
@gdams apline jdk, jre tests failed. Could you please take a look at?

File: linux/jdk/redhat/src/main/packaging/build.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ echo "DEBUG: building RH arch ${buildArch} with jdk version ${buildVersion}"
 if [ "${buildArch}" != "all" ]; then
 	targets=${buildArch}
 elif [ "${buildVersion}" = "20" ]; then
```
Comment (andrew-m-leonard) [inline linux/jdk/redhat/src/main/packaging/build.sh:34]: guessing this should be = "21" ?

Comment (sophia-guo) [reply linux/jdk/redhat/src/main/packaging/build.sh]: Yes, updated.

File: linux/jdk/alpine/src/main/packaging/temurin/21/APKBUILD [hunk 2]
```diff
@@ -1,11 +1,9 @@
 # Maintainer: Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>
-pkgname=temurin-21
-pkgver=21.0.0_p35
+pkgname=temurin-21.0.1+12
```
Comment (gdams) [inline linux/jdk/alpine/src/main/packaging/temurin/21/APKBUILD:2]:
```suggestion
pkgname=temurin-21
```

File: linux/jre/alpine/src/main/packaging/temurin/21/APKBUILD [hunk 3]
```diff
@@ -1,11 +1,9 @@
 # Maintainer: Eclipse Adoptium Package Maintainers <temurin-dev@eclipse.org>
-pkgname=temurin-21
-pkgver=21.0.0_p35
+pkgname=temurin-21.0.1+12
```
Comment (gdams) [inline linux/jre/alpine/src/main/packaging/temurin/21/APKBUILD:2]:
```suggestion
pkgname=temurin-21
```

**Review by gdams — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]

**Pull request merged**